src/test/java/http/RequestHandlerTest.java

```diff
@@ -0,0 +1,49 @@
+package http;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class RequestHandlerTest {
+    RequestHandler requestHandler = new RequestHandler();
+
+    @Test
+    public void getRequestTest() throws IOException {
+        String request = "GET /simple_get HTTP/1.1\r\n";
```

Comment:
Tweak: Some repetition, consider making it a public var

Reply:
I have tweaked the test cases to improve their quality.